integration-test/src/it/simple-numeric-directory-path-inverse/verify.groovy

```diff
@@ -53,6 +53,6 @@ buildLog.eachLine {
 }
 assert executionRoot == true
 
-File json = new File( basedir, 'target/manipulation.json' )
+File json = new File( basedir, 'target/alignmentReport.json' /* + ManipulationManager.REPORT_JSON_DEFAULT*/ )
```

Comment:
Couldn't figure out how to use constant `ManipulationManager.REPORT_JSON_DEFAULT` (not on classpath?).

Reply:
They are isolated tests so its likely not on classpath

Reply:
Yeah, if I wanted to reference it, I wasn't sure which POM to add it to. I tried it in the `simple-numeric-directory-path-inverse/pom.xml`, but it didn't seem to work.

Reply:
They are being run by https://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html , specifically https://maven.apache.org/plugins/maven-invoker-plugin/examples/post-build-script.html